README.md

```diff
@@ -42,17 +42,55 @@ Some websites using this theme:
 
 ## Installation
 
-To use the theme in your documentation, copy the required CSS and JS files from this repository into your project or add the repository as submodule and check out the latest release:
+To use the theme when generating your documentation, bring the required CSS and JS files from this repository into your project.
+
+This can be done in several ways:
+
+- manually copying the files
+- adding the project as a Git submodule
+- adding the project as a npm/xpm dependency
+- installing the theme system wide
+
+All theme files are located in the root of this repository and start with the prefix `doxygen-awesome-`. You may not need all of them. Follow the install instructions to figure out what files are required for your setup.
+
+### Git submodule
+
+For projects which use git, add the repository as a submodule and check out the desired release:
 
 ```bash
 git submodule add https://github.com/jothepro/doxygen-awesome-css.git
 cd doxygen-awesome-css
-git checkout v2.2.0
+git checkout v2.2.1
 ```
 
-You can even install the theme system-wide by running `make install`. The files will be installed to `/usr/local/share/` by default, but you can customize the install location with `make PREFIX=/my/custom/path install`.
+### npm dependency
 
-All theme files are located in the root of this repository and start with the prefix `doxygen-awesome-`. You may not need all of them. Follow the install instructions to figure out what files are required for your setup.
+In the npm ecosystem, this project can be added as a development dependency
+to your project:
+
+```bash
```

Comment:
Since this isn't a bash shell script but just a set of commands to put into a terminal this should probably be a `plaintext`  block instead.

Reply:
I can put any block type there, if necessary, but what is the difference between a shell script and a sequence of shell commands, and when should one be used over the other?

I generally use `bash` for blocks which define shell commands and `console` for blocks which define commands and their output.

Reply:
> I can put any block type there, if necessary, but what is the difference between a shell script and a sequence of shell commands, and when should one be used over the other?

This is a personal opinion, but generally unless I really want to specify a shell script I just use `plaintext` because it doesn't make the text look weird with some random highlighted words like `cd` . Also, using `bash` leaves you vulnerable to questions like "why `bash` and not plain `sh`?", but yeah, this is mostly just personal preference.

Reply:
I changed them to `sh`, to be more generic, but probably this has no impact on how the markdown code clock is rendered in html.